cuplaConfig.cmake

```diff
@@ -22,7 +22,7 @@
 # Required cmake version.
 ################################################################################
 
-cmake_minimum_required(VERSION 3.11.0)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.11.4)
```

Comment:
lowercase is sexy, modern, and consistent with the other CMake code below:
```suggestion
cmake_minimum_required(VERSION 3.11.4)
```

Reply:
copy past copy past copy past^^